packages/components/.stylelintrc.js

```diff
@@ -0,0 +1,150 @@
+    // TO DISCUSS:
+    // =============================
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://stylelint.io/user-guide/rules/list/alpha-value-notation/
+    'alpha-value-notation': null,
+
+    // not sure what to do with this rule, seems the default is "never" with some exceptions
+    // probably I would keep the default provided by 'stylelint-config-standard-scss'
+    // see: https://stylelint.io/user-guide/rules/list/at-rule-empty-line-before/
+    // 'at-rule-empty-line-before': null,
+
+    // preference for 'legacy'
+    // see: https://stylelint.io/user-guide/rules/list/color-function-notation/
+    'color-function-notation': null,
+
+    // would disable this rule, impacts a "grid-template-areas" definition, not sure why
+    // see: https://stylelint.io/user-guide/rules/list/declaration-block-no-redundant-longhand-properties/
+    'declaration-block-no-redundant-longhand-properties': null,
+
+    // what option here? 'never-single-line' or 'never'?
+    // see: https://stylelint.io/user-guide/rules/list/function-parentheses-space-inside/
+    'function-parentheses-space-inside': 'never-single-line',
+
+    // I propose to use "2" here (in some cases is useful to add extra visual space between blocks to better separate/group content)
+    // see: https://stylelint.io/user-guide/rules/list/max-empty-lines/
+    // 'max-empty-lines': null,
+    'max-empty-lines': 2,
+
+    // see: https://stylelint.io/user-guide/rules/list/max-line-length/
+    // TODO: I can't get the regex to work, I'll get back to this later
+    // 'max-line-length': [200, { ignore: ['non-comments'], ignore: ['comments'], ignorePattern: ["/.*; //.*/"] }],
+    'max-line-length': null,
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://stylelint.io/user-guide/rules/list/number-leading-zero/
+    'number-leading-zero': null,
+
+    // no strong opinions, probably would keep what we have now (no big impact)
+    // see: https://stylelint.io/user-guide/rules/list/number-max-precision/
+    'number-max-precision': 5,
+
+    // I would prefer to be explicit when a mixin is called
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-mixin-argumentless-call-parentheses/README.md
+    'scss/at-mixin-argumentless-call-parentheses': 'always',
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-rule-conditional-no-parentheses/README.md
+    'scss/at-rule-conditional-no-parentheses': null,
+
+    // not sure why we would want this rule active as is by default, probably needs some tweaking (so my suggestion is to simply disable it)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/double-slash-comment-empty-line-before/README.md
+    'scss/double-slash-comment-empty-line-before': null,
```

Comment:
I wouldn't want to enforce a rule on this. 

I think what makes sense depends so there should be flexibility.